### DISCUSSION TITLE
Using MSW for testing crashes the app, but not the tests themselves

### DISCUSSION THREAD

**Discussion by nathan-hyan** [mswjs/msw#922] [Q&A]
Hello everyone!

I'm running into this issue where i can use MSW for testing and everything works just fine! until i try to run the React app, where it throws the next error:

![image](https://user-images.githubusercontent.com/45558924/134239817-c8928c8a-0d71-455e-a7fa-87421c095086.png)

My Typescript version is 4.4.2 and MSW version is the latest.

**Comment by kettanaito** [comment]
Hi, @nathan-hyan. 

It looks like a TypeScript issue. We're currently using TypeScript 4.3.5 in MSW and have no guarantees that it functions properly in 4.4.x. The update to the latest TypeScript version is in progress in #930 and has not been released yet. 

For now, I can only recommend suppressing the error message until the mentioned dependency update is finished. Thanks for understanding.
